app/src/main/res/values/styles.xml

```diff
@@ -8,8 +9,32 @@
         <item name="android:textAllCaps">true</item>
         <item name="android:textColor">@color/yellow_main_text</item>
     </style>
-    <style name="ConfigureParkingSizeButtonMainActivityStyle">
+
+    <style name="ParkingSizeButtonMainActivityStyle">
         <item name="android:backgroundTint">@color/orange_parking_size_button_background</item>
         <item name="android:textColor">@color/dark_parking_size_button_text</item>
     </style>
+
+    <style name="ParkingButtonDialogFragmentStyle">
+        <item name="android:backgroundTint">@color/orange_parking_size_button_background</item>
+        <item name="android:textColor">@color/dark_parking_size_button_text</item>
+        <item name="android:layout_marginTop">@dimen/margin_top_parking_size_button</item>
+    </style>
+
+    <style name="TextParkingSizeDialogFragmentStyle">
+        <item name="android:fontFamily">sans-serif-bold</item>
+        <item name="android:textSize">@dimen/size_dialog_parking_size_text</item>
+        <item name="android:gravity">center</item>
+    </style>
+
+    <style name="InputParkingSizeDialogFragmentStyle">
+        <item name="android:inputType">number</item>
+        <item name="android:textAlignment">center</item>
+        <item name="android:layout_marginTop">@dimen/margin_top_main_activity_title</item>
+        <item name="android:layout_width">0dp</item>
```

Comment:
the android:layout_width should be in the view not in the style

Reply:
Done